PKGBUILD

```diff
@@ -0,0 +1,63 @@
+pkgname='ros-melodic-franka-ros'
+pkgver='0.6.0'
+pkgrel=2
```

Comment:
pkgrel should be 1

Reply:
Is it really that big problem? For me it is a second version of the package, as I have changed the source from release to base repository, and in order to test the build on my build machine I had to bump the version.